docs/getting-started/namespace.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+kind: Namespace
```

Comment:
Nice! Should fix the integration tests!

Reply:
Hmm, looks like we have to ensure this is is the first YAML that gets applied.
An alternative might be to explicitly add the create namespace in `test/e2e-tests-ingress.sh`

Reply:
I renamed the ns yaml to `01-namespace.yaml`, I wonder if it will get ordered first.

Reply:
That works too!